contracts/test/v0.8/dev/KeeperRegistryDev.test.ts

```diff
@@ -538,15 +538,6 @@ describe('KeeperRegistryDev', () => {
             emptyBytes,
           )
         id = await getUpkeepID(tx)
-
-        await registry
-          .connect(owner)
-          .registerUpkeep(
-            mock.address,
-            executeGas,
-            await admin.getAddress(),
-            emptyBytes,
-          )
```

Comment:
not sure if this is intended to be part of this PR?

Reply:
yes. i realized that i accidentally register 1 more upkeep than this test needs.